### DISCUSSION TITLE
Google開發出AI機器學習模型！ 幫助地圖上的商家「自動更新」營業時間

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#425] [科技新知]
「Google開發了一種機器學習模型，這個模型可以自動識別營業時間是否可能出錯，再使用AI生成的預測來更新營業時間。」
這個ai模型可以偵測店家實際營業時間並更新，還可以判斷是否打電話給店家確認，實在為店家省掉許多麻煩。

資料來源：https://www.google.com/amp/s/tw.news.yahoo.com/amphtml/google%25E9%2596%258B%25E7%2599%25BC%25E5%2587%25BAai%25E6%25A9%259F%25E5%2599%25A8%25E5%25AD%25B8%25E7%25BF%2592%25E6%25A8%25A1%25E5%259E%258B-%25E5%25B9%25AB%25E5%258A%25A9%25E5%259C%25B0%25E5%259C%2596%25E4%25B8%258A%25E7%259A%2584%25E5%2595%2586%25E5%25AE%25B6-%25E8%2587%25AA%25E5%258B%2595%25E6%259B%25B4%25E6%2596%25B0-%25E7%2587%259F%25E6%25A5%25AD%25E6%2599%2582%25E9%2596%2593-080100723.html

**Comment by YuHengChen54** [comment]
這功能感覺很棒欸，不然有時候真的會在google地圖上看有開，結果到現場沒開。

**Comment by lx3211** [comment]
街景圖像是會一直更新的嗎？
因為常常看到的照片都是很舊的
很好奇到底是用什麼資料來判斷是否營業

**Comment by brian431** [comment]
走到想吃的餐廳，也確認過營業時間結果發現沒開。真的是非常可怕的一件事情

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by xCk27x** [comment]
但是google地圖的更新頻率要足夠高
這項服務的可信度才會提升
不然也有可能發生店家重新營業卻被判斷未開門的情況
到時候造成店家困擾就不好了QQ

**Comment by NcuJerryKao** [comment]
很有效率 對大眾不錯的應用

**Comment by Ben6uo** [comment]
這樣可以節省很多時間 非常方便我們的使用

**Comment by todoroki0910** [comment]
這個功能確實該被快點落實，不然不知道要被耍幾次

**Comment by gnsJhenJie** [comment]
過去(好幾年前)幫忙更新Google Maps上的資訊或評論是可以累積點數得到Google Drive額外空間的，現在可能資料量已經非常足夠，所以沒有這個獎勵了。現在只有可以累積「在地嚮導」等即獲得徽章而已(但聽說等級到超高的Google還是會給一些實體獎品)

**Comment by skyling0299** [comment]
太好了，不然常常被耍QQ

**Comment by BingRong1217** [comment]
對社會大眾助益相當大！

**Comment by limitx0** [comment]
我都不會完全相信Google地圖上的營業時間
但是很期待這項AI能讓營業時間準確到什麼地步

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by awkward-willy** [comment]
透過這樣的功能能讓資訊的更新更即時，也能減少人力在更新這些資訊的浪費。不過感覺這比較適合做為一個提醒或備用手段，最好還是仰賴店家自主更新自己的資訊，畢竟在這個時代，人人都會上網看評價、營業時間等，若店家不自主更新，對消費者、店家都是雙輸的局面。
